molecule/default/roles/helm/defaults/main.yml

```diff
@@ -9,7 +9,7 @@ tiller_cluster_role: cluster-admin
 chart_test: "nginx-ingress"
 chart_test_version: 1.32.0
 chart_test_version_upgrade: 1.33.0
-chart_test_repo: "https://kubernetes-charts.storage.googleapis.com"
+chart_test_repo: "https://charts.helm.sh/stable"
```

Comment:
We may not want to use this repository because it will be deprecated soon.
See: https://github.com/ansible-collections/community.kubernetes/pull/294#issuecomment-723246774

Reply:
I thought https://charts.helm.sh/stable/ is the new repo?

Reply:
https://github.com/helm/charts/blob/master/README.md#deprecation-timeline

Reply:
This being said, it seems the repository will stay available. It won't be updated any-more. This is not a big deal in our case. So @Akasurde, just ignore my message and let's get this CI fixed ASAP.